### ISSUE TITLE
LibreOffice plugin: problem with italicized text in title 

### ISSUE THREAD

**Issue by michaelpalmeruw** [JabRef/jabref#2483]
JabRef version 3.7 on Debian Jessie

Steps to reproduce:

1. Open a LibreOffice document. 
2. Select a bibliography style that italicizes the whole title, such as one of the two default styles that come with JabRef itself.
2. Cite a reference whose title uses some words, <i>some italicized words</i>, and some non-italicized ones. (Use `<i>some italicized words</i>` to apply italics.)
3. The output in the reference list will be: "_some words, some italicized words,_ and some non-italicized ones." 

In other words, a local `</i>` tag __terminates__ the global application of italics to the entire title. Edit: This also applies if the JabRef record contains `\emph{some italicized words}`, and analogously with `<b>` tags and `\textbf{some bold-faced words}`, although the latter would be a less common scenario (I just tested for curiosity). But italics is practically important, as it is used for example for biological species names.

**Comment by michaelpalmeruw** [comment]
I should have been more careful with those HTML italics tags. I had typed them in but the web app swallowed them. So, to be clear, I had < i > and < / i > to apply italics formatting, and it is the < / i > that closes both the local and the global application of italics.

**Comment by Siedlerchr** [comment]
@michaelpalmeruw  Thanks for your report. To avoid problems with automatic parsing of html or other code here, you can wrap such thinks in code-tags by selecting the text and then hitting the code  (the <> button)

**Comment by michaelpalmeruw** [comment]
Thanks, Christoph. I updated my post accordingly.

**Comment by stefan-kolb** [comment]
The problem should be located inside `OOUtil.java#insertOOFormattedTextAtCurrentLocation`. Looks like the method cannot handle nested tags in general.

**Comment by stefan-kolb** [comment]
It might be inside the `OOpreformatter.java`.

**Comment by stefan-kolb** [comment]
Inside `OOUtil#insertOOFormattedTextAtCurrentLocation`

**Comment by stefan-kolb** [comment]
Thank you for your report :+1:
This should be fixed in current `oo-formatting`. Please try the latest build from http://builds.jabref.org/oo-formatting.

**Comment by stefan-kolb** [comment]
@michaelpalmeruw Thank you for your report 👍
This should be fixed in current oo-formatting. Please try the latest build from http://builds.jabref.org/oo-formatting.

**Comment by michaelpalmeruw** [comment]
@stefan-kolb - yes, the fix works fine. Thank you very much.
